File: C/Program05.c
/////////////////////////////////////////////////////////////////////////
//                                                                     //
//  File name :     Program5.c                                         //
//  Description :   Used to print natural numbers in reverse order     //
//  Author :        Pallavi Omprakash Malewar                          //
//  Date :          28/04/2025                                         //
//                                                                     //
/////////////////////////////////////////////////////////////////////////

#include<stdio.h>

// Iteration 

void Display(int iNumber)
{
    int iCount = 0;

    for(iCount = iNumber ; iCount >= 1 ; iCount--)
    {
        printf("%d\n" , iCount);
    }
}


int main()
{
    int iValue = 0;

    printf("Enter the Value : ");
    
    scanf("%d",&iValue);

    Display(iValue);

    return 0;
}

File: JAVA/Program05.java
////////////////////////////////////////////////////////////////////////////////////
//                                                                                //
//  File name :     Program05.java                                                //
//  Description :   Used to Display Total Count of Factors of Given Number        //
//  Author :        Pallavi Omprakash Malewar                                     //
//  Date :          24/08/2025                                                    //
//                                                                                //
////////////////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class Number
{
    public int totalFactors(int iNo)
    {
        int iCount = 0, iCnt = 0;

        for(iCount = 1 ; iCount <= (iNo / 2) ; iCount++)
        {
            if((iNo % iCount) == 0)
            {
                iCnt++;
            }
        }
        iCnt++;             // To Add Count of Given Number itself
        return iCnt;
    }
}


class Program05
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0 , iRet = 0 ;

        System.out.print("Enter Number : ");
        iValue = sobj.nextInt();

        Number nobj = new Number();

        iRet = nobj.totalFactors(iValue);

        System.out.println("Total Number of Factors of " + iValue + " is "+ iRet);

        sobj.close();

    }
}